ci/deploy.sh

```diff
@@ -62,18 +66,37 @@ cp -f "$FIRST_FILE" "$FPATH.jar"
 
 ###### Build the deploy command ######
 MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -DretryFailedDeploymentCount=3 -B"
+DEPLOY_CMD="$MVN -B deploy:deploy-file -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
+    -DgroupId=com.nvidia -DartifactId=spark-rapids-jni -Dversion=$REL_VERSION -s ci/settings.xml"
+SIGN_CMD='eval nvsec sign --job-name "Spark Jar Signing" --description "Sign artifact with 3s"'
```

Comment:
The `eval` here is odd, and if truly necessary a comment would help clarify why this is needed.

Reply:
`eval`  treat the Env SIGN_CMD string containing quotations interpreted as one cmd, otherwise ￥SIGN_CMD is interpret as `nvsec sign --job-name Spark Jar Signing --description Sign artifact with 3s`

As `eval` is not recommended, I will find other ways to fix the problem

Reply:
IMHO the most straightforward way to solve this is to create a function that does the signing.  For example:
```bash
function sign_jar() {
  nvsec sign --job-name "Spark Jar Signing" --description "Sign artifact with 3s" "$@"
}

[...]

sign_jar --file $FPATH.jar --out-dir $OUT_PATH
```

Reply:
Good suggestion, let me update it.